netutils/utils_solaris.go

```diff
@@ -1,10 +1,11 @@
-// Package ipamutils provides utililty functions for ipam management
-package ipamutils
+package netutils
```

Comment:
ping @amitkris Can you please check this is correct. This was because of a recent change.